src/App.js

```diff
@@ -1,6 +1,6 @@
 import "./App.css";
 import { Route, Routes } from "react-router-dom";
-import {Home, Header, Footer, Productlist} from "./components/import.js"
+import {Home, Header, Footer, Productlist, Cart} from "./components/import.js"

```

Comment:
You can rename this file to index.js so you don't have to write components/filename here